phoenix-core/src/it/java/org/apache/phoenix/end2end/TableDDLPermissionsIT.java

```diff
@@ -210,25 +98,20 @@ public Void run() throws Exception {
                     return null;
                 }
             });
-            verifyAllowed(createSchema(schemaName), regularUser);
+            verifyAllowed(createSchema(schemaName), regularUser1);
             // Unprivileged user cannot drop a schema
-            verifyDenied(dropSchema(schemaName), unprivilegedUser);
-            verifyDenied(createSchema(schemaName), unprivilegedUser);
+            verifyDenied(dropSchema(schemaName), AccessDeniedException.class, unprivilegedUser);
+            verifyDenied(createSchema(schemaName), AccessDeniedException.class, unprivilegedUser);
 
-            verifyAllowed(dropSchema(schemaName), regularUser);
+            verifyAllowed(dropSchema(schemaName), regularUser1);
         } finally {
             revokeAll();
         }
     }
 
     @Test
-    public void testAutomaticGrantDisabled() throws Throwable{
-        testIndexAndView(false);
-    }
```

Comment:
Why is this removed?

Reply:
I removed the option of AUTOMATIC_GRANT and set it defaulted to true.